<launch>
  <node pkg="turtlebot_maze" type="obstacle_avoidance_srv" name="obstacle_avoidance_node"  output="screen">
  </node>

  <node pkg="turtlebot_maze" type="main_control" name="main_control_node" 
  output="screen">
  </node> 
</launch>